### PR TITLE
Update test dataset to include more sample images with better resolut…

### LINKED ISSUE #44: Bigger sample dataset for ymir-imaging
We want to cover all the angles, maybe every 10 degrees and slightly better resolution.

Just need to update the sample dataset.

### PR DESCRIPTION
**Pull request by YooSunYoung**
Fixes #44 

Now it has 36 sample images covering every 10 degrees of rotation angle
with image size 256x256.

Data analysis workflow needed more rotation angles to test their methods.

The file size is ~6 Mb.

The file was tested in the #42

### CODE REVIEW

File: src/ess/imaging/data.py [hunk 1]
```diff
@@ -15,8 +15,8 @@ def _make_pooch():
         base_url='https://public.esss.dk/groups/scipp/ess/imaging/',
         version=_version,
         registry={
-            'small_ymir_images.hdf': 'md5:5fbc52595f3e6117a2be2bb50866806a',
-            'README.md': 'md5:6ca6b4a79bcff2965a81767e12474617',
+            'small_ymir_images.hdf': 'md5:cf83695d5da29e686c10a31b402b8bdb',
+            'README.md': 'md5:0f375972d4008de6060b065ac13ba17f',
```
Comment (nvaytet) [inline src/ess/imaging/data.py:19]: I'm curious: why is the `README.md` in there?

Comment (YooSunYoung) [reply src/ess/imaging/data.py:19]: It's a readme of the dataset.
It also has a code snippet how the dataset was made.

Comment (MridulS) [reply src/ess/imaging/data.py:19]: Hmm, where is the readme file? https://public.esss.dk/groups/scipp/ess/imaging/, am I looking at the wrong place?

Comment (YooSunYoung) [reply src/ess/imaging/data.py:19]: It is there.
image


Maybe the web browser interface is only showing the files with certain extensions?

Comment (MridulS) [reply src/ess/imaging/data.py:19]: ah yeah true.

**Review by MridulS — APPROVED** [review]

**Pull request merged**